### ISSUE TITLE
Suggestion - Tag/Hash extension

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1364]
Hi, 

Tagging means to attach a label to a record. You can then search tags through Global Search and manage tags in the new Tags Module.   

My team and me are thinking that will very usefull if we can use the tag function/extension for our work

The work processes will faster, better and easier, when everybody will have some helper to group the record by Tags/Hash

Now we have feature function(module), but it is usefull only if you have less records,

We saw this and it is amazing... https://goo.gl/gQdf3e

What does the community think about it?

**Comment by fgpalacios** [comment]
Is an excellent feature.

Regards ,

Fernando.-

**Comment by pgorod** [comment]
I'm currently using the tagMe module with my SuiteCRM and I am happy with it.

**Comment by Mausino** [comment]
I think, here is total description how the tag function can work 

http://support.sugarcrm.com/Documentation/Sugar_Versions/7.7/Ent/Application_Guide/Tags/

My opinion is that as others parst(modules), this is also very important part of any CRM system, because i am using tags in any important aplication for me like gmal, keep, evernote, thunderbird, outlook, joomla! etc. Tags help me many time save the time and buid custom "the self user logic" by user itself.
.
@shogunpol @JimMackin  @ijdavie @murdal @gregsoper what are you thinking ?

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Suggestion - Tag/Hash extension](https://trello.com/c/U6XwkTpV/99-suggestion-tag-hash-extension)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
